README.md

```diff
@@ -79,24 +79,19 @@ specifying different package names.
 
 ### Docker
 
-Users wishing to use Eland without installing it, in order to just run the available scripts, can build the Docker
-container:
+Users wishing to use Eland without installing it, in order to just run the available scripts, can use the Docker
+image.
+It can be used interactively:
```

Comment:
I know it's not your sentence originally, but we can improve it now if you don't mind:
```suggestion
If you want to use Eland without installing it just to run the available scripts, use the Docker
image.
It can be used interactively:
```